readthedocs/core/utils/__init__.py

```diff
@@ -179,6 +179,10 @@ def prepare_build(
     # Log all the extra options passed to the task
     structlog.contextvars.bind_contextvars(**options)
 
+    # NOTE: call this log here as well to log all the context variables added
+    # inside this function. This is useful when debugging.
+    log.info("Triggering build with specific attributes.")
```

Comment:
This is an odd message? The comment kind of explains it, but we should probably make the message make more sense in the context of the logs?